code/debris/debris.cpp

```diff
@@ -430,7 +430,7 @@ object *debris_create(object *source_obj, int model_num, int submodel_num, vec3d
 		if (vaporize) {
 			dist /= 2.0f;
 		}
-		if ( dist > 200.0f ) {
+		if ( dist > source_obj->radius * 200.0f ) {
```

Comment:
Is this right?  Shouldn't it be addition?

Reply:
Ah! That was a typo, I should have 20.0f there, not 200. 10m fighter -> 1m debris, 20.0f x 10m = 200m. so it would roughly follow the precedent set by the original hardcoded value.